examples/fabric-bridge-app/linux/RpcServer.cpp

```diff
@@ -160,6 +162,52 @@ pw::Status FabricBridge::ActiveChanged(const chip_rpc_KeepActiveChanged & reques
     return pw::OkStatus();
 }
 
+pw::Status FabricBridge::AdminCommissioningAttributeChanged(const chip_rpc_AdministratorCommissioningChanged & request,
+                                                            pw_protobuf_Empty & response)
+{
+    NodeId nodeId = request.node_id;
+    ChipLogProgress(NotSpecified, "Received CADMIN attribut change: " ChipLogFormatX64, ChipLogValueX64(nodeId));
+
+    auto * device = BridgeDeviceMgr().GetDeviceByNodeId(nodeId);
+    if (device == nullptr)
+    {
+        ChipLogError(NotSpecified, "Could not find bridged device associated with nodeId=0x" ChipLogFormatX64,
+                     ChipLogValueX64(nodeId));
+        return pw::Status::NotFound();
+    }
+
+    BridgedDevice::AdminCommissioningAttributes adminCommissioningAttributes;
+
+    uint32_t max_window_status_value =
+        static_cast<uint32_t>(chip::app::Clusters::AdministratorCommissioning::CommissioningWindowStatusEnum::kUnknownEnumValue);
+    VerifyOrReturnValue(request.window_status < max_window_status_value, pw::Status::InvalidArgument());
+    adminCommissioningAttributes.commissioningWindowStatus =
+        static_cast<chip::app::Clusters::AdministratorCommissioning::CommissioningWindowStatusEnum>(request.window_status);
+    if (request.has_opener_fabric_index)
+    {
+        VerifyOrReturnValue(request.opener_fabric_index >= chip::kMinValidFabricIndex, pw::Status::InvalidArgument());
+        VerifyOrReturnValue(request.opener_fabric_index <= chip::kMaxValidFabricIndex, pw::Status::InvalidArgument());
+        adminCommissioningAttributes.openerFabricIndex = static_cast<FabricIndex>(request.opener_fabric_index);
+    }
+    else
+    {
+        adminCommissioningAttributes.openerFabricIndex.reset();
+    }
+
+    if (request.has_opener_vendor_id)
+    {
+        VerifyOrReturnValue(request.opener_vendor_id != chip::VendorId::NotSpecified, pw::Status::InvalidArgument());
+        adminCommissioningAttributes.openerVendorId = static_cast<chip::VendorId>(request.opener_vendor_id);
+    }
+    else
+    {
+        adminCommissioningAttributes.openerVendorId.reset();
```

Comment:
Same as above, openerVendorId field hasn't been modified before calling reset() in the scenario where request.has_opener_vendor_id is false.